api/tests/test_token_metadata_processor.py

```diff
@@ -26,13 +26,15 @@ async def main():
 
     result = tokenMetadataProcessor.get_default_token_metadata(registryAddress='0x57E9a39aE8eC404C08f88740A9e6E306f50c937f',tokenId=165)
     expected = RetrievedTokenMetadata(registryAddress='0x57E9a39aE8eC404C08f88740A9e6E306f50c937f',tokenId='165',metadataUrl=base64.b64encode('{}'.encode()),imageUrl=None,name='#164',description=None,attributes=[])
-    #assert (result == expected)
+    assert (result == expected)
 
-    result = await tokenMetadataProcessor.retrieve_token_metadata(registryAddress='0x4581649aF66BCCAeE81eebaE3DDc0511FE4C5312', tokenId='6574')
+    collection = RetrievedCollection(address='0x4581649aF66BCCAeE81eebaE3DDc0511FE4C5312', name='TheAlienBoy', symbol='TABOY', description='Each Alien Boy is unique and algorithmically generated by combining 182 properties with varying rarities in 9 categories (Backgrounds, Skin, Face, Eyes, Body, Mouth, Accessory, Head, Special Power)\nLimited supply of 10.000 Alien Boys invading the shitty earth. Our alien-source brand grants ownership and commercial usage rights given to the owner not only over their NFT but also to The Alien Boy open source brand.   \n\n*Give us some SPACE in your wallet, human*   \n\n*One Alien Boy in your wallet is a curiosity 👽  Two are an invasion 👽 🛸 👽*\n\n[The Alien Girl Collection](https://opensea.io/collection/thealiengirl)    \n[The Alien UFO Collection](https://opensea.io/collection/thealienufo)  \n[The Alien Doogle](https://opensea.io/collection/thealiendoogle)  ', imageUrl='https://lh3.googleusercontent.com/mnNWlPXlDTp8N1JP2kiCrTy-w0lGU0S1AX5sMm0ZhWjSTpIh4RTwXERNjz83aKI1uKjyqf8yZJlsId6TeUgvJRhwaWScJPdWZSlISJc=s120', twitterUsername='TheAlienBoyNFT', instagramUsername=None, wikiUrl=None, openseaSlug='thealienboy', url='https://www.thealienboy.com/', discordUrl='https://discord.gg/thealienboy', bannerImageUrl='https://lh3.googleusercontent.com/bWgbzHdOrgOtpDUm0LTIo4eH4JjzeDUsTce4qA3mrBkGw-wXYnWDQVxaKXBG28Y2kQ634cAFBYEfsBMpqggMirS9Vu6GKxAOeKnqJ00=s2500', doesSupportErc721=True, doesSupportErc1155=False)
```

Comment:
This should be a Collection not RetreivedCollection

Reply:
also you can truncate the description - its a unit test so we can make up input data here it doesnt need to be exact

Reply:
What do you mean by truncate?

Reply:
shorten, like just use 2 words instead of the whole text